data/tribes/initialization/empire/units.lua

```diff
@@ -1138,6 +1142,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_headquarters_shipwreck",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Headquarters Shipwreck
+            lore = pgettext("empire_building", "‘Naufragium sed domo.’ (Although it's wrecked, it's still our home)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Headquarters Shipwreck
+            lore_author = pgettext("Lutius after stranding on some uncharted Island"),
```

Comment:
```suggestion
            lore_author = pgettext("Lutius after stranding on an uncharted island"),
```
IMHO Lutius should here be named with a title like General or Commander or … (don't remember if his title is ever mentioned in the campaign)

Reply:
Scenario 1 says he commanded 150 soldiers when he was sent to the Barbarian frontier. Does that make him a centurio?

OTOH exactly because he is never addressed by any rank or title in the campaign, I think it is best to not start it here.

Reply:
Lutius is a young general in the imperial army 
see
https://www.widelands.org/wiki/EmpireStorySummary/

Reply:
Oops. :) (though a young general is kind of a contradiction)

But still, why should we call him General Lutius here, when he's just Lutius all over the campaign?

Reply:
in the Campaign he is adressed as Lutius only by his sister Amalea all other People are adressing him with Sire!   
but here we talk about him in third person and I believe in this case his rank is part of his name.

Reply:
His name alone is also used for talking about him in 3rd person in titles, like "Diary of Lutius" in all 4 scenarios, and in emp04:

- "Lutius is disappointed"
- "Lutius greets the official"
- "Lutius accepts the challenge"
- translator hint: "Marcus - Mayor of Fremil welcoming Lutius"